### PR TITLE
Score Page redesigned and Practice option incuded in the Signup Page

### PR DESCRIPTION
**Pull request by pranatsharma1**

### CODE REVIEW

**Review by deepanshu-jain1999 — COMMENTED** [review]
1. Optimized your commit name(not signified the changes)
2. Follow PEP8(here there is no newline in the end of the file)

File: core/static/core/css/instruction.css [hunk 1]
```diff
@@ -195,3 +230,15 @@ margin:10px 0 0 30px !important;
...
+  {
+    width: 100% !important;
+  }
+  .margin
+{
+  margin-top:25vh !important;
+  margin-left: 0vw !important;
+}
+  
+}
```
Comment (deepanshu-jain1999) [inline core/static/core/css/instruction.css]: #newline

**Comment by deepanshu-jain1999** [comment]
@harshs14 @Stark-Pramod @Sharma-palak @pranatsharma1 @ritik09 @mahima-c   You guys can use Black for the python projects, It will automatically done those changes that are required to be done for PEP8 and helps in write better code in style

We can easily install it in pycharm as a plugin

**Comment by Pramodjais517** [comment]
> @harshs14 @Stark-Pramod @Sharma-palak @pranatsharma1 @ritik09 @mahima-c You guys can use Black for the python projects, It will automatically done those changes that are required to be done for PEP8 and helps in write better code in style
> 
> We can easily install it in pycharm as a plugin

we will try it for sure sir.

**Comment by pranatsharma1** [comment]
> @harshs14 @Stark-Pramod @Sharma-palak @pranatsharma1 @ritik09 @mahima-c You guys can use Black for the python projects, It will automatically done those changes that are required to be done for PEP8 and helps in write better code in style
> 
> We can easily install it in pycharm as a plugin

Sure Sir.

**Pull request merged**